test/JDBC/input/BABEL-736.sql

```diff
@@ -0,0 +1,95 @@
+SELECT degrees(10)
```

Comment:
1. Please move everything in this file into the upgrade test and use proc/view there (ref: https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/876)
2. We should have all tests where input parameters are any data types that can be implicitly casted into numeric expression, including const number and literal.